packages/shared/src/hooks/useInputFieldFunctions.ts

```diff
@@ -41,7 +41,7 @@ function useInputFieldFunctions<
   const [validInput, setValidInput] = useState<boolean>(undefined);
   const [idleTimeout, clearIdleTimeout] = useDebounceFn(() => {
     setValidInput(inputRef.current.checkValidity());
-  }, 1500);
+  }, 500);
```

Comment:
I have changed this to 500 because I noticed when the URL is invalid the error message below the input box shows a bit later and feels a little delayed in 1500. Maybe you can test it and let me know if I should keep this 500 or 1500 same as before.

Reply:
Yeah, I agree, 1500ms can probably be too much. IIRC, the debounce we have on search is 500, too.